src/libs/ReportUtils.js

```diff
@@ -777,9 +759,8 @@ function getIcons(report, personalDetails, defaultIcon = null) {
             return [result];
         }
 
-        const actorEmail = lodashGet(parentReportAction, 'actorEmail', '');
+        const actorEmail = parentReportAction.actorEmail;
```

Comment:
`lodashGet` is okay here if we want to fallback into an empty string instead of `undefined`.